torch/functional.py

```diff
@@ -981,6 +984,114 @@ def cdist(x1, x2, p=2., compute_mode='use_mm_for_euclid_dist_if_necessary'):
     else:
         raise ValueError("{} is not a valid value for compute_mode".format(compute_mode))
 
+def atleast_1d(*tensors):
+    r"""
+atleast_1d(input) -> Tensor
+
+View input tensors to tensor with at least one dimension.
+
+Scalar input tensors are converted to 1-dimensional arrays, whilst higher-dimensional inputs are preserved.
+
+Args:
+    input (Tensor or list of Tensors): the input tensor.
```

Comment:
I don't think you need to describe input; you can probably remove ": the input tensor."